packages/paste-core/components/slider/stories/index.stories.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from 'react';
+import {Label} from '@twilio-paste/label';
+import {HelpText} from '@twilio-paste/help-text';
+import {useUID} from '@twilio-paste/uid-library';
+import {Form, FormControl} from '@twilio-paste/form';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {useTheme} from '@twilio-paste/theme';
+
+import {Slider} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Slider',
+};
+
```

Comment:
suggestion (non-blocking): can you add a story where the currently selected value is visible somewhere on the screen?